packaging/greenboot/functions.sh

```diff
@@ -135,6 +137,21 @@ function get_wait_timeout() {
     echo "${wait_timeout}"
 }
 
+# Exit early if previous MicroShift healthcheck scripts failed.
+function exit_early_if_previous_checks_failed() {
```

Comment:
Nit: to be consistent in naming, the function does not "know" if it's an early or late exit.
Should we rename to something like: `exit_if_fail_marker_exists`?